elements/rh-subnav/rh-subnav.css

```diff
@@ -83,7 +83,7 @@ button {
   background-color: transparent;
   border: none;
   color: var(--_subnav-next-prev-button-text-color, #6a6e73);
-  --pfe-icon--size: 16px;
+  --pfe-icon--size: 14px;
```

Comment:
should use pf-icon apis at this point

Reply:
> should use pf-icon apis at this point

Question for @coreyvickery and @marionnegp the design spec (for tabs) shows 14px height on the icon.  I was matching that style here, as we don't currently have a spec for subnav overflow, it just mocks tabs.

Reply:
@coreyvickery @zeroedin, I think 10px looks better. An 18px icon looks too big with the tab text.